server/controllers/personSearch/personSearchByCRN.ts

```diff
@@ -4,7 +4,7 @@ import config from '../../config'
 export const personSearchByCRN = async (req: Request, res: Response): Promise<Response | void> => {
   res.locals.notification = {
     ...config.notification,
-    isVisible: config.notification.body && config.notification.active,
+    isVisible: Boolean(config.notification.active === 'true'.toLowerCase()) && Boolean(config.notification.body),
```

Comment:
'true'.toLowerCase() is redundant.

You've just explicitly made it lowercase.

The enclosing Boolean is redundant as well.  

If you want you can use the ! or !! as a boolean converter.  But what you've done here is fine.

ie.  !!config.notification.body

Reply:
Thanks! All resolved!